src/contolers/users.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+const userServise = require('../services/users.js');
+
+let nextId = 0;
+
+function add(req, res) {
+  const { name } = req.body;
+
+  if (!name) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newUser = {
+    id: nextId++,
+    name,
+  };
+
+  userServise.add(newUser);
+
+  res.statusCode = 201;
+  res.send(newUser);
+};
+
+function getAll(req, res) {
+  const users = userServise.getAll();
+
+  res.statusCode = 200;
+
+  if (!users) {
+    res.send([]);
+
+    return;
+  }
```

Comment:
is getAll return always array?
```suggestion
  const users = userServise.getAll();

  res.statusCode = 200;
```